### ISSUE TITLE
Programs always output REPL-like output after normal output

### ISSUE THREAD

**Issue by duckyluuk** [RockstarLang/rockstar2#4]
Currently, a program always outputs an evaluation of the last executed line below the normal output:
![image](https://github.com/user-attachments/assets/5b73b222-3038-438a-98e8-05f9dc5557be)
(the above example using a program simply being:
```
shout "Hello World!"
```

This makes sense when in the interactive mode, but when executing a program from a file this makes a lot less sense. Take the following example program:
```
the index = 0
the word = "test"
while the index is less than 4
say the word at the index
build the index up
```
The output is:
![image](https://github.com/user-attachments/assets/22ca0692-89a6-41ed-b82f-2b450afeb9fe)
where the `« 4` does not make much sense.

I'd suggest leaving the eval output out of executed programs, while it can be left the same for the interactive mode.

**Comment by dylanbeattie** [comment]
Yep, agree 100%. I'll put it on the "list of things to fix before shipping" :)

**Comment by Yewzir** [comment]
@dylanbeattie
How do we temporarily eliminate this ourselves? We're really just eager to do some thorough testing in order to help improve the project.

Regards
@Yewzir

**Comment by dylanbeattie** [comment]
Fixed in https://github.com/RockstarLang/rockstar2/commit/0daad170e0d25ea6857447e1bcafda9909a9d759

https://github.com/RockstarLang/rockstar2/releases/tag/v0.11.2

**Comment by Yewzir** [comment]
> Fixed in [0daad17](https://github.com/RockstarLang/rockstar2/commit/0daad170e0d25ea6857447e1bcafda9909a9d759)
> 
> https://github.com/RockstarLang/rockstar2/releases/tag/v0.11.2

Thank you.
